queenbee/env/os.py

```diff
@@ -0,0 +1,18 @@
+import platform
+
+
+class OS:
+    """A simple wrapper to handle OS-specific switches.
+
+    Attributes:
+        is_windows: A boolean of whether the current system is MS Windows.
+        is_nix: A boolean of whether the current system is *nix (any Linux).
+        file_uri_prefix: A string to be used as the default prefix for URIs
+            which point to local files on the system.
+
+    """
+
+    is_windows: bool = platform.system() == 'Windows'
```

Comment:
This attribute not being read-only can be tricky but since this is a simple wrapper I think we are good in this case.